include/cpp-statsd-client/StatsdClient.hpp

```diff
@@ -67,6 +67,7 @@ class StatsdClient {
 
     StatsdClient(const StatsdClient&) = delete;
     StatsdClient& operator=(const StatsdClient&) = delete;
+    StatsdClient(StatsdClient&&) = default;
```

Comment:
fixes a warning in windows builds

Reply:
Why `default` and not `delete`?

Reply:
turns out that was only a warning when i turned on `/Wall` which was way too far of a level to turn on anyway so i can just remove it